resources/public/api/conf/1.0/examples/checkpoints/list_response.json

```
@@ -0,0 +1,48 @@
+{
+  "checkpoints": [
+    {
+      "checkpointId": "a1c8057e-fbbc-47a8-a8b4-78d9f015c253",
+      "nino": "TC663795B",
+      "checkpointCreationTimestamp": "2019-07-13T07:51:43.112Z",
+      "links": [
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints/a1c8057e-fbbc-47a8-a8b4-78d9f015c253/restore",
+          "method": "POST",
+          "rel": "restore-checkpoint"
+        },
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints/a1c8057e-fbbc-47a8-a8b4-78d9f015c253",
+          "method": "DELETE",
+          "rel": "delete-checkpoint"
+        },
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints?nino={nino}",
+          "method": "POST",
+          "rel": "create-checkpoint"
+        }
+      ]
+    },
+    {
+      "checkpointId": "b2e4050e-fbbc-47a8-d5b4-65d9f015c253",
+      "nino": "RC972995B",
+      "checkpointCreationTimestamp": "2019-07-13T08:51:43.112Z",
+      "links": [
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints/a1c8057e-fbbc-47a8-a8b4-78d9f015c253/restore",
+          "method": "POST",
+          "rel": "restore-checkpoint"
+        },
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints/a1c8057e-fbbc-47a8-a8b4-78d9f015c253",
+          "method": "DELETE",
+          "rel": "delete-checkpoint"
+        },
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints?nino={nino}",
```

I think we typically exclude unknown query params from HATEOAS links
```suggestion
          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints",
```